config_defaults_inc.php

```diff
@@ -1179,6 +1179,13 @@
  */
 $g_datetime_picker_format = 'Y-MM-DD HH:mm';
 
+/**
+ * Datetime picker widget format string for date only.
+ * For detailed instructions on date formatting
+ * @see http://momentjs.com/docs/#/displaying/format/
+ * @global string $g_datetime_picker_format
+ */
+$g_date_picker_format = 'Y-MM-DD';
```

Comment:
Is it possible to use short date format or a transformation of it rather than adding this?  If we are going to add this, it should be added to the manual.